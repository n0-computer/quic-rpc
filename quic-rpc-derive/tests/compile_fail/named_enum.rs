use quic_rpc_derive::rpc_requests;

#[rpc_requests(Service)]
enum Enum {
    A { name: u8 },
}

fn main() {}