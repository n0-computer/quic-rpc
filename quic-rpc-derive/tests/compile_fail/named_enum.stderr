error: Each variant must have exactly one unnamed field
 --> tests/compile_fail/named_enum.rs:5:5
  |
5 |     A { name: u8 },
  |     ^
